### DISCUSSION TITLE
People cant access my github account and repositories

### DISCUSSION THREAD

**Discussion by dzikryaji** [community/community#75326] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Just like the title my account cannot be access by other people. i can still log in to my account and see my repositories as usual but other account cannot find my account. first, i think my account get suspended but i realised that i didn't receive any email from github. So is there anything i can do to fix this problem? thx for your answers and help

btw this is my account link [https://github.com/dzikryaji]()

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @dzikryaji! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've asked @queenofcorgis to move this issue into the correct category for you. Good luck!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @dzikryaji ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
